packages/rke2-calico/generated-changes/patch/values.yaml.patch

```diff
@@ -41,3 +41,10 @@
 +
 +global:
 +  systemDefaultRegistry: ""
++
++ipamConfig:
++  strictAffinity: true
++  autoAllocateBlocks: true
++
++felixConfiguration:
++  allowVXLANPacketsFromWorkloads: true
```

Comment:
this defaults to false and is not required to for strict affinity

Reply:
Ok. Then I suggest leaving the default